pkg/kubelet/events/event.go

```diff
@@ -34,6 +34,11 @@ const (
 	FailedToCreatePodContainer     = "FailedCreatePodContainer"
 	FailedToMakePodDataDirectories = "Failed"
 	NetworkNotReady                = "NetworkNotReady"
+	ResizeInProgress               = "ResizeInProgress"
+	ResizeDeferred                 = "ResizeDeferred"
+	ResizeInfeasible               = "ResizeInfeasible"
+	ResizeError                    = "ResizeError"
+	ResizeCompleted                = "ResizeCompleted"
```

Comment:
nit: Please rename ResizeCompleted -> ResizeComplete

Reply:
And it can be invoked if s.Resize is empty.
https://github.com/kubernetes/kubernetes/blob/6cb5ea56cb96f6a4ddec677d3d94faf07179368d/pkg/kubelet/kubelet_pods.go#L1796-L1798